ui/src/tempVars/components/TemplatePreviewList.tsx

```diff
@@ -3,42 +3,38 @@ import uuid from 'uuid'
 
 import {ErrorHandling} from 'src/shared/decorators/errors'
 import FancyScrollbar from 'src/shared/components/FancyScrollbar'
+import TemplatePreviewListItem from 'src/tempVars/components/TemplatePreviewListItem'
+import {TEMPLATE_PREVIEW_LIST_DIMENSIONS as DIMENSIONS} from 'src/tempVars/constants'
 
-const LI_HEIGHT = 28
-const LI_MARGIN_BOTTOM = 2
-const RESULTS_TO_DISPLAY = 10
+import {TemplateValue} from 'src/types'
+
+const {RESULTS_TO_DISPLAY, LI_HEIGHT, LI_MARGIN_BOTTOM} = DIMENSIONS
```

Comment:
I think these could be factored out in favor of using some props on `FancyScrollbar`. If you set `AutoHeight={true}` it will expect a `maxHeight={number}` to go with it. The maximum height should be a number that results in the last visible item in the list to be partially obscured, which helps inform the user that the list extends beyond the visible frame. I suppose it's a question of whether this belongs in the JS or CSS domain